app/queries/servers/post.ts

```diff
@@ -138,6 +138,10 @@ export const queryPostsBetween = (database: Database, earliest: number, latest:
         andClauses.push(Q.where('root_id', rootId));
     }
 
+    if (isCRTEnabled) {
+        andClauses.push(Q.where('root_id', ''));
+    }
+
```

Comment:
Is there any case were both rootID and isCRTEnabled are defined in this function? If so, does it work well with two clauses being "where rootId = rootId" and "where rootId=''"?